src/views/VideoView/VideoView.tsx

```diff
@@ -31,6 +31,24 @@ const VideoView: React.FC<RouteComponentProps> = () => {
 
   const videoID = data?.video?.id
 
+  const [playing, setPlaying] = useState<boolean>(true)
+  const handleUserKeyPress = useCallback((event) => {
+    const { key, keyCode } = event
```

Comment:
Once we have `event` typed we can do `event as KeyboardEvent` here to get access to the data we need